quickwit/quickwit-proto/src/lib.rs

```diff
@@ -252,3 +252,19 @@ impl search::SortOrder {
 }
 
 impl quickwit_common::pubsub::Event for ReportSplitsRequest {}
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize)]
+pub struct PipelineMetrics {
+    pub cpu_thousandth: u16,
```

Comment:
Looking at other codebases (Systemd, Linux Kernel, Kube), looks like `cpu_shares` is more commonly used with the `millicore` unit.

Reply:
cool i did not know that word!